README.md

```diff
@@ -0,0 +1,32 @@
+# Getting Started with webaverse-blockchain-lib
+
+```sh
+npm install
+```
+
+```sh
+npm run prepare
+```
+
+### cd examples
+
+```sh
+npm run start
+```
+
+# About the project
+There are four functions(Login with metamask, Login with discord, Fetch Profile and Update the name) in this project.
+
+1.  Login with metamask
+    - Connect metamask
+    
+    ![1](https://user-images.githubusercontent.com/87816136/162272579-0de10c04-d6ad-4dab-8761-a662adb6baf5.png)
+    - Login to metamask
+    
+    ![2](https://user-images.githubusercontent.com/87816136/162272746-19bafc67-e028-489c-adce-b851cccd4655.png)
+2.  Login with discord
+    The function of login with discord is not implemented yet.
+3.  Fetch Profile
+    By clicking “Fetch Profile” you can get data of wallet and your personal information.
```

Comment:
click what?  please clearly seperate the documentation for the library, and the examples